rest-api/src/main/java/gov/cms/qpp/conversion/api/controllers/v1/CpcFileControllerV1.java

```diff
@@ -98,13 +100,36 @@
 			headers = {"Accept=" + Constants.V1_API_ACCEPT} )
 	public ResponseEntity<String> markFileProcessed(@PathVariable("fileId") String fileId) {
 		if (blockCpcPlusApi()) {
-			API_LOG.info("CPC+ unprocessed files request blocked by feature flag");
+			API_LOG.info(BLOCKED_BY_FEATURE_FLAG);
 			return new ResponseEntity<>(null, null, HttpStatus.FORBIDDEN);
 		}
 
 		API_LOG.info("CPC+ update file as processed request received");
 		String message = cpcFileService.processFileById(fileId);
-		API_LOG.info("CPC+ update file as processed request succeeded");
+		API_LOG.info("CPC+ update file as processed request succeeded with message: " + message);
+		HttpHeaders httpHeaders = new HttpHeaders();
+		httpHeaders.setContentType(MediaType.TEXT_PLAIN);
+
+		return new ResponseEntity<>(message, httpHeaders, HttpStatus.OK);
+	}
+
+	/**
+	 * Updates a file's status to unprocessed in the database
+	 *
+	 * @param fileId Identifier of the file needing to be updated
+	 * @return Message if the file was updated or not
+	 */
+	@RequestMapping(method = RequestMethod.POST, value = "/unprocess/{fileId}",
```

Comment:
Mmmm...  I'm not the biggest fan of `POST /cpc/unprocess/{fileId}` to mark a file as unprocessed since it uses a verb in the URL and also given the existing way of marking a file as processed (`PUT /cpc/file/{fileId}`).

What about these following ideas?
- `PUT /cpc/file/{fileId}` and the request body can contain...
  ```
  {
      "status": "<processed/unprocessed>"
  }
  ```
  The `UnprocessedCpcFileData` class can possibly be retrofitted for the above request body.
- `PUT /cpc/file/{fileId}?processed` and `PUT /cpc/file/{fileId}?unprocessed`.
- A slight tweak to the above is `PUT /cpc/file/{fileId}?status=<processed/unprocessed>`.

Reply:
I'm a proponent of the approach using `PUT /cpc/file/{fileId}` along with a request body.  That's much more RESTful.  Technically, a PUT should contain _all_ of the fields for the object, but no one really uses `PATCH` for partial updates.

Reply:
Good ideas, thanks for the feedback.